src/index.js

```diff
@@ -81,6 +83,6 @@ export { default as AeSwitch } from './components/aeSwitch/aeSwitch.vue'
 export { default as AeAppIcon } from './components/aeAppIcon/aeAppIcon.vue'
 export { default as AeIcon } from './components/aeIcon/aeIcon.vue'
 export { default as AeAddress } from './components/aeAddress/aeAddress.vue'
-export { default as AeAddressInput } from './components/aeAddressInput/aeAddressInput.vue'
```

Comment:
Why you have to remove AeAddressInput component?

Reply:
:(